src/features/pricing/PricingPageContainer.tsx

```diff
@@ -15,6 +19,10 @@ type ModalState =
     };
 
 const PricingPageContainer: FunctionComponent = () => {
+  const { t } = useTranslation();
+
+  const { loading, error, data } = useQuery<PRICING_PAGE>(PRICING_PAGE_QUERY);
```

Comment:
Should we prevent caching for this query (always fetch)? Now it's never fetched again until page reload / relogin. It's likely that multiple admins are updating the prices.